functions.js

```diff
@@ -1 +1,82 @@
-// YOUR CODE HERE
+/*
+sillySum(arr)
+Write a function that takes an array of numbers and returns the sum of each number multiplied by its index.
+count += (number * index)
+*/
+
+function sillySum(arr){
+  var result = arr.reduce(function multByIndex(a, b) {
+    return a + (b * arr.indexOf(b)); });
+  return result;
+}
+
+sillySum([2, 2, 3, 4, 5]);
+
```

Comment:
off by 1. you don't multiply the first value by it's index.  The loop variation works